.circleci/config.yml

```diff
@@ -17,17 +17,8 @@ jobs:
       - run: |
           source $NVM_DIR/nvm.sh
```

Comment:
Tbh, this is kind of dodgy.
I'll try to find a more reliable alternative. 
Maybe [avn](https://github.com/wbyoung/avn)?